client/admin/api/user.js

```diff
@@ -1,46 +1,45 @@
 import { extractData, processParams } from '@/common/api/helpers';
 import get from 'lodash/get';
+import path from 'path';
 import request from '@/common/api/request';
 
-const url = {
+const urls = {
   root: '/users',
-  resource: it => `/users/${it.id}`,
-  invite: it => `/users/${it.id}/invite`,
-  import: '/users/import',
-  getImportTemplate: '/users/import/template'
+  resource: id => path.join(urls.root, String(id)),
+  invite: id => path.join(urls.root, String(id), 'invite'),
+  import: () => path.join(urls.root, 'import'),
+  getImportTemplate: () => path.join(urls.root, 'import, template')
```

Comment:
`, ` personally feels less readable.

Reply:
I agree in some cases. But I think `path.join` is safer since it normalizes backslashes.
Also it's semantically more accurate since we're talking about route **paths**.

Reply:
In general I do agree, I am referring to this exact case and import/template.

Reply:
Since we're talking about import, I think I will be good idea to decouple import from user in a separate PR.
By doing that we'll have generic common import and it'll be reusable for any entity. Thoughts? 🤔